src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1,4 +1,95 @@
 featurizers:
+  # ----------------------------------------------------------------------
+  - name: AnalyticalRollingWindowFeaturizer
+    estimator_name: AnalyticalRollingWindowEstimator
+    release_version: 0.4.0
+    num_output_columns: 1
+    has_dynamic_output: false
+    description: |-
+        Calculates data based on a rolling window. Currently supports mean. Works for general time series data sets.
+        
+        Input type for this featurizer is a tuple of time point of current row, the grain columns, and target value column.

```

Comment:
>Input type for this featurizer is a tuple of time point of current row, the grain columns, and target value column. [](start = 8, length = 115)

This description needs to be updated now that we are no longer taking a datetime as input.

Reply:
Good catch. I have removed that from the description and added that we are assuming the data is already sorted in the correct order.

Reply:
Good catch. I have removed that from the description and added that we are assuing the data is already sorted in the correct order.